package.json

```diff
@@ -9,7 +9,8 @@
     "test": "tape src/tests/test.js",
     "lint": "eslint ./src",
     "lint-fix": "eslint --fix ./src",
-    "eslint-init": "eslint --init ./src"
+    "eslint-init": "eslint --init ./src",
+    "db:query": "psql -U postgres -h localhost -d hmcqc -a -f ./query.sql"
```

Comment:
There's a problem during running this command 
```psql -U  "database-user"  "database-name" -a -f ./query.sql```

Reply:
Done